src/CSnakes.SourceGeneration/PythonStaticGenerator.cs

```diff
@@ -150,4 +148,19 @@ public interface I{{pascalFileName}} : IDisposable
             }
             """;
     }
+
+    private static string HexString(ReadOnlySpan<byte> bytes)
```

Comment:
`Convert.ToHexString` should do this, no?

Reply:
It's [only available since .NET 5](https://learn.microsoft.com/en-us/dotnet/api/system.convert.tohexstring), not for .NET Standard 2.0, which is what the source generator is targeting:

https://github.com/tonybaloney/CSnakes/blob/efd03f84eb3bac396d1bcd5408cdfc31b5b04333/src/CSnakes.SourceGeneration/CSnakes.SourceGeneration.csproj#L3